tests/e2e/utils/context.go

```diff
@@ -29,14 +29,14 @@ type TestContext struct {
 	Cfg Config
 }
 
-//function to get the testcontext Object.
+//fNewTestContext unction to get the testcontext Object.
```

Comment:
fNewTestContext  -> NewTestContext
//also the comment can be improved like :  NewTestContext builds a new test context with the provided configuration.

Reply:
done